frontend/src/components/MeetingRoom/VideoOutputDevices/VideoOutputDevices.tsx

```diff
@@ -8,7 +8,6 @@ import Popper from '@mui/material/Popper';
 import { PopperChildrenProps } from '@mui/base';
 import VideoDevices from './VideoDevices';
 import VideoDevicesOptions from './VideoDevicesOptions';
-import displayOnDesktop from '../../../utils/displayOnDesktop';
```

Comment:
Note: after this `import` was removed I noticed that `displayOnDesktop` util is not being used anywhere else in the codebase so I decided to remove it. Recently we moved more to the concept of small view port devices anyways besides just differentiating between mobile/desktop.